src/Doctrine/ReadModel/CacheDocumentRepository.php

```diff
@@ -48,7 +48,8 @@ public function save(JsonDocument $document, int $attempts = 3): void
 
         $savedDocument = $this->fetch($document->getId());
         if ($savedDocument->getRawBody() !== $document->getRawBody()) {
-            $this->logger->error(
+            $this->logger->log(
+                $attempts > 1 ? 'warning' : 'error',
```

Comment:
Shouldn't it also be `warning` if there is still 1 attempt left? I'd expect an error if the last attempt fails